### PR TITLE
Add Sqxtn_S, Sqxtn_V, Uqxtn_S, Uqxtn_V instructions and Tests (6).

### PR DESCRIPTION
**Pull request by LDj3SNuD**
Update Tester as well.
Nits.

### CODE REVIEW

**Comment by gdkchan** [comment]
Thanks for implementing this! The implementation seems correct, the only problem is that it seems to use EmitLdint/Stint to load/store temp values. However the Ldint/Stint methods are used to read/write to the general purpose registers, so this instruction is modifying the values of the GPRs (X0/X1/X2).

You can use Ldinttmp/Stinttmp to load/store to a temp variable, however the emitter only exposes one temp. I'd recommend writing it in a way that only uses 1 or no temps.

Also, more descriptive names to the labels (like LblEnd or LblExit at the end of the function) facilitates the understanding.

**Comment by LDj3SNuD** [comment]
Updated.
(Ldinttmp/Stinttmp do not exist, it would be useful to be able to use them.)

**Comment by gdkchan** [comment]
Sorry, the correct (re: Ldinttmp/Stinttmp) is EmitLdtmp/EmitSttmp.

File: ChocolArm64/Instruction/AInstEmitSimdArithmetic.cs [hunk 1]
```diff
@@ -205,6 +205,79 @@ private static void EmitHighNarrow(AILEmitterCtx Context, Action Emit, bool Roun
...
+                Context.Emit(OpCodes.Pop);
+
+                Context.EmitLdc_I4(TMinValue);
+                SetQCFlag();
+
+                Context.MarkLabel(LblGeEnd);
+
+                if (Scalar)
+                {
+	                EmitVectorZeroLower(Context, Op.Rd);
```
Comment (gdkchan) [inline ChocolArm64/Instruction/AInstEmitSimdArithmetic.cs]: nit: Seems to have one extra identation level.

**Comment by LDj3SNuD** [comment]
ReUpdated & ReTested.

**Comment by gdkchan** [comment]
lgtm, thanks!

**Pull request merged**